src/lib/containers/row_renderers/Study.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { Chip } from './utils/index';
+
+const STUDY_SCHEMA = {
+    projectName :0,
+    id : 1,
+    projectFileviewId : 2,
+    projectStatus :3,
+    dataStatus : 4,
+    fundingAgency : 5,
+    summary : 6,
+    summarySource : 7,
+    projectLeads : 8,
+    institutions : 9,
+    tumorType : 10,
+    diseaseFocus: 11
+}
+
+const CUTOFF = 250
+
+export default class Study extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            showMore: false
+        }
+        this.toggleShowMore = this.toggleShowMore.bind(this)
+    }
+
+    toggleShowMore(event) {
+        event.preventDefault()
+        this.setState({
+            showMore: !this.state.showMore
+        })
+    }
+
+    render () {
+        const {data} = this.props
+        const projectName = data[STUDY_SCHEMA.projectName]
+        const projectLeads = data[STUDY_SCHEMA.projectLeads] && data[STUDY_SCHEMA.projectLeads].split(";").join(" / ")
+        let summary = data[STUDY_SCHEMA.summary]
+
+        const diseaseFocus = <Chip type="gray" text={data[STUDY_SCHEMA.diseaseFocus]}></Chip>
+        const tumorType = <Chip type="blue" text={data[STUDY_SCHEMA.tumorType]}></Chip>
+        
+        const projectStatus = data[STUDY_SCHEMA.projectStatus]
+        const fundingAgency = data[STUDY_SCHEMA.fundingAgency]
+        const dataStatus = data[STUDY_SCHEMA.dataStatus]
+        const institutions = data[STUDY_SCHEMA.institutions]
+        return <div className="container SRC-syn-border SRC-noPaddingBottom  SRC-syn-border-spacing">
```

Comment:
Agree with you, I think reusable Util components should be used in favor of this...